File: chap15_muscle/exercises/assay1.m


clear all
close all
clc

syms G V0 V1 V2

V = V0 + G*V1;

wun = 1 - 2*G*V*(1+V/G);
cc = collect(simplify(wun),G);
cf = coeffs(cc,G);
cf1 = simplify(cf(1))
V0sol = solve(cf1==0,V0)
cf2 = simplify(subs(cf(2),V0,V0sol))
V1sol = solve(cf2==0,V1)

File: chap15_muscle/exercises/assay1.py
from sympy import *

V0,V1,G = symbols("V0 V1 G")

V = V0 + V1*G
pp = 1 - (2*V*G)*(1)*(1+V/G)
out = Poly(series(pp,G,0,2).removeO(),G)

cc = out.coeffs()
print(cc)
V0val = solve(cc[1],V0)[1]
print(V0val)

dum = cc[0].subs(V0,V0val)
print(solve(dum,V1))
